crashtracker/src/rfc5_crash_info/error_data.rs

```diff
@@ -0,0 +1,64 @@
+use std::collections::HashMap;
```

Comment:
just out of curiosity, shouldn't this be after the header ?

Reply:
Yes.  OOPS